src/foobar/admin.py

```diff
@@ -192,6 +192,25 @@ def changelist_view(self, request, extra_context=None):
         return response
 
 
+@admin.register(models.WalletLogEntry)
+class WalletLogEntryAdmin(admin.ModelAdmin):
+    list_display = ('date_created',
+                    'user',
+                    'trx_type',
+                    'post_balance',
+                    'amount',
+                    'pre_balance'
+                    )
+    ordering = ('-date_created',)
+
```

Comment:
You should add a Meta-class here and specify `verbose_name` and `verbose_name_plural`.